index.html

```diff
@@ -62,7 +62,7 @@
     <header id="hero">
       <div class="navbar">
         <div></div>
-        <h1 class="logo"><a href="#home"><img src="./src/assets/images/chryz-hub-transparent.png" alt="Logo-image"></a></h1>
+        <h1 class="logo text-primary"><a href="#home"><img src="./src/assets/images/chryz-hub-transparent.png" alt="Logo-image"></a></h1>
```

Comment:
This `text-primary` class shouldn't be added to the logo heading. This is the output. Will fix it though. Thanks @AmjadNoor 

![remove](https://user-images.githubusercontent.com/62628408/116937634-733b3f80-ac61-11eb-8c40-0234b23ae28f.png)